modular_skyrat/modules/cyborg/code/robot_upgrades.dm

```diff
@@ -244,3 +244,41 @@
 		var/obj/item/inducer/cyborg/inducer = locate() in target_robot.model
 		if (inducer)
 			target_robot.model.remove_module(inducer, TRUE)
+
+/obj/item/borg/upgrade/sign
+	name = "Cyborg sign waving upgrade"
+	desc = "A sign unit for cyborgs, allowing them to easily advertise products through an intuitive, wave-able sign."
+	icon_state = "cyborg_upgrade3"
+	require_model = TRUE
+	model_type = /obj/item/robot_model/service
+
+/obj/item/borg/upgrade/sign/action(mob/living/silicon/robot/target_robot, user = usr)
+	. = ..()
+	if(!.)
+		return
+	var/obj/item/picket_sign/cyborg/borgsign = locate() in target_robot.model.modules
+	if(borgsign)
+		target_robot.balloon_alert(user, "already equipped!")
+		return
+
+	borgsign = new(target_robot.model)
+	target_robot.model.basic_modules += borgsign
```

Comment:
Why do you add it to the basic modules? I'm not very familiar with cyborg code, so I figured I'd ask.